src/Supportive/DependencyInjection/Configuration.php

```diff
@@ -180,6 +180,18 @@ private function appendFormatters(ArrayNodeDefinition $node): void
                                 })
                             ->end()
                         ->end()
+                        ->arrayNode('mermaidjs')
+                            ->info('Configure MermaidJS output formatter')
```

Comment:
Can you please add `->addDefaultsIfNotSet()` here to fix #949 similar to graphviz and codeclimate (see #951)